src/Sum/README.md

```diff
@@ -0,0 +1,118 @@
+# Sum
+```haskell
+Sum Number
+```
+`Sum` is a `Monoid` that will combine two numbers using Addition.
+
+```js
+const Sum = require('crocks/Sum')
+const mconcat = require('crocks/helpers/mconcat')
+const mconcatMap = require('crocks/helpers/mconcatMap')
+
+Sum(13)
+//=> Sum 13
+
+mconcat(Sum, [5,2])
+//=> Sum 7
+
+Sum(10)
+  .concat(10)
+//=> Sum 20
+
+// mapToSumByTen :: [Number] -> Sum
+const mapToSumByTen = mconcatMap(Sum, x => x * 10)
+
+mapToSumByTen([2,2])
```

Comment:
may be easier to read with a smaller name, spaces and some other bit so it reads like (also the result is not right):
```js
// sumByTen :: [ Number ] -> Sum
const sumByTen =
  mconcat(Sum, x => x * 10)

sumByTen([ 2, 2 ])
//=> Sum 40
```